File: src/monster/player.js
const express = require('express');
const {  players } = require('../data/database');
const router = express.Router();
const uniqueMiddleware = require("../validator/uniqueMiddleware");
router.get("/players", (req, res) => res.json(players));

router.post("/player", uniqueMiddleware(players), (req, res) => {
    const { name, attack, health } = req.body;
    players.push({
        name,
        attack,
        health,
        status: health > 0 ? 'alive' : 'death',
    });

    res.json(players);
})
module.exports = router;

File: Exercise 4/Gamer/Player.java
package Gamer;

public class Player {
    private String name;
    private char key;

    public Player(String name, char key) {
        this.name = name;
        this.key = key;
    }

    public String getName() {
        return name;
    }

    public char getKey() {
        return key;
    }
}
